### DISCUSSION TITLE
Now it just works?

### DISCUSSION THREAD

**Discussion by kejid** [frondeus/fvtt-syrin-control#114] [General]
Just installed 0,5.2 of this module, added token and apparently now, with foundry v11 v315 it simply plays inside game as well as through separate syrinscape app. Kinda amazed and confused, because this functionality is nowhere mentioned

**Comment by frondeus** [comment]
Hi, sorry I'm not sure if I understand correctly, what functionality is not mentioned? Audio playback in Foundry?

**Reply by kejid** [reply]
Yeah, after importing soundset from syrinscape and pressing play, music comes within vtt. No external player is required. So I was just amazed by that, because originally it was suppose to be replication of control panel, just inside foundry

**Reply by frondeus** [reply]
Oh I see, that changed in 0.4.0 :) 
And I definitely mentioned it :D https://github.com/frondeus/fvtt-syrin-control#audio--volume-control

But I'm glad that you like it!

**Reply by kejid** [reply]
Yeah, my bad I missed it) Still, cool mod anyway, I remember 2 years ago it was kinda painful to setup using csv, now it is much more convenient =)
